cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc

```diff
@@ -0,0 +1,17 @@
+[id="cnv-monitoring-vm-health"]
+= Monitoring virtual machine health
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-monitoring-vm-health
+toc::[]
+
+Liveness probes stop the `VirtualMachineInstance` if it fails and allow
+controllers such as `VirtualMachine` or `VirtualMachineInstanceReplicaSet` to spawn
+other `VirtualMachineInstances`, in order to restore responsiveness to the
+virtual machine.
```

Comment:
I think this sentence is too long/wordy. Potential suggestion (I'm not sure if you need both examples or not):

"When a VirtualMachineInstance (VMI) fails, _liveness probes_ stop the VMI. Controllers such as `VirtualMachine` then spawn other VMIs, restoring virtual machine responsiveness."

(note that I think "liveness probes" should have emphasis since we're introducing them here -- correct me if I'm wrong about that)

Reply:
Done.